MarsRover.Tests/InputParserShould.cs

```diff
@@ -20,7 +19,7 @@ public void ParseDirection_ReturnsDirection_GivenValidString(string input, Direc
         {
             var result = InputParser.ParseDirection(input);
 
-            Assert.Equal(expected, result);
+            result.Should().Equals(expected);

```

Comment:
Seems like Fluent Assertions documents `Should.Be()` for Enums. https://fluentassertions.com/enums/